plugins/SitesManager/lang/en.json

```
@@ -191,6 +191,18 @@
         "SiteWithoutDataStartTrackingDataDescriptionLine1": "Choose your preferred tracking method below. If you need help, you can share these %1$sinstructions via email%2$s or %3$sinvite another user%4$s for assistance.",
         "SiteWithoutDataStartTrackingDataDescriptionLine2": "Note: This screen will disappear as soon as some data is tracked for this website. Alternatively, you can %1$sclick here to hide this page for one hour%2$s.",
         "EmailInstructionsButtonText": "Email instructions",
-        "SiteWithoutDataIgnorePage": "Don't show this page for an hour"
+        "SiteWithoutDataIgnorePage": "Don't show this page for an hour",
+        "SiteWithoutDataVue": "Vue",
+        "SiteWithoutDataVueDescription": "You can use the \"%1$s\" npm package to integrate Matomo to track data, follow the instructions from this %2$sguide%3$s.",
+        "VueDetected": "Vue was detected on your website. Did you know you can use the \"%1$s\" npm package to integrate Matomo into your site? %2$sLearn more in this guide%3$s.",
+        "SiteWithoutDataVueIntro": "Tracking your Vue website or app is easy with Matomo, even for Single Page Applications.",
+        "SiteWithoutDataVueFollowStep1": "Install the %1$s npm package.",
```

Might be more consistent to have the package name in quotes here as well.
```suggestion
        "SiteWithoutDataVueFollowStep1": "Install the \"%1$s\" npm package.",
```

---

Added as suggested